agents-cred/src/main/java/org/apache/ranger/authorization/credutils/CredentialsProviderUtil.java

```diff
@@ -91,7 +81,7 @@ public static KerberosCredentialsProvider getKerberosCredentials(String user, St
 
     public static synchronized KerberosTicket getTGT(Subject subject) {
         Set<KerberosTicket> tickets = subject.getPrivateCredentials(KerberosTicket.class);
-        for(KerberosTicket ticket: tickets) {
+        for (KerberosTicket ticket : tickets) {
             KerberosPrincipal server = ticket.getServer();
             if (server.getName().equals("krbtgt/" + server.getRealm() + "@" + server.getRealm())) {
                 if (logger.isDebugEnabled()) {
```

Comment:
Most of the occurrences of logger.isDebugEnabled() may be removed, please check.